tests/docs/running-e2e-test.md

```diff
@@ -90,8 +95,14 @@ make docker-build
 # Push docker image to your dockerhub registry
 make docker-push
 
+# Replace version placeholders in helm configuration files
+sh .github/scripts/set_helm_dapr_version.sh
+
 # Deploy Dapr runtime to your cluster
 make docker-deploy-k8s
+
+# Revert version placeholders in helm config files
+sh .github/scripts/revert_helm_dapr_version.sh
```

Comment:
`git checkout` would revert changes, still need this?

Reply:
When debugging locally, because you need to manually replace `DAPR_VERSION_HELM` and `DAPR_VERSION_TAG`, after debugging, you need to revert the change. In these operations, git checkout will not be used

Reply:
Change version string -> debug -> revert, changes to those files can be git checkout to the original.

Reply:
If you change some content under the `charts` pkg, git checkout is not necessarily a very good behavior, because it will revert the records you really want to change.It is not the reverse of `set_helm_dapr_version.sh`
@daixiang0 What do you think?

Reply:
Let's use one script `set_helm_dapr_version.sh`, add a param to revert/set.

Reply:
> Let's use one script `set_helm_dapr_version.sh`, add a param to revert/set.

@daixiang0 Done